src/main/java/org/agileandbeyond/dandd/tddexercise/Race.java

```diff
@@ -0,0 +1,147 @@
+package org.agileandbeyond.dandd.tddexercise;
+
+public class Race {
+
+	private RaceType raceType;
+
+	public Race() {
+		// Race defaults to Human
+		this.raceType = RaceType.HUMAN;
+	}
+
+	public RaceType getRaceType() {
+		return raceType;
+	}
+
+	public void setRaceType(RaceType raceType) {
+		this.raceType = raceType;
+	}
+
+	public int getAbilityModifier(AbilityType abilityType) {
+		int abilityModifier = 0;
+
+		switch (this.raceType) {
+		case HUMAN:
+			abilityModifier = getHumanModifiers(abilityType);
+			break;
+		case ORC:
+			abilityModifier = getOrcModifiers(abilityType);
+			break;
+		case DWARF:
+			abilityModifier = getDwarfModifiers(abilityType);
+			break;
+		case ELF:
+			abilityModifier = getElfModifiers(abilityType);
+			break;
+		case HALFLING:
+			abilityModifier = getHalflingModifiers(abilityType);
+			break;
+		}
+		return abilityModifier;
+	}
+
+	private int getHumanModifiers(AbilityType AbilityType) {
```

Comment:
Surely we should have some advantage for being human??!!  Maybe intelligence or something?  ;)